rules/game_process.tex

```diff
@@ -340,6 +416,53 @@ \subsection{Free Kick}
 Note that in the case of the Pushing free kick the sub-state is activated automatically through the ``Foul''.
 The team who is awarded the free kick (termed the attacking team) has \qty{\FreeKickTime}{\second} to complete the kick.
 
+\subsubsection{Visual gesture}
+\label{sec:free_kick_gesture}
+
+\textbf{In Champions Cup only}, the referee communicates which team is to take the free kick via a visual gesture.
+The GameController packet will tell the robots which free kick sub-state is active, but not who is to kick the ball.
+
+\begin{figure*}[t]
+  \centering
+  \begin{subfigure}{.33\textwidth}
+      \centering
+      \includegraphics[height=120px]{figs/visual-signal/kick-in.png}
+      \caption{Pointing to the left goal}
+  \end{subfigure}
+  \begin{subfigure}{.33\textwidth}
+      \centering
+      \includegraphics[height=120px]{figs/visual-signal/kick-in-flipped.png}
+      \caption{Pointing to the right goal}
+  \end{subfigure}
+  \caption{Visual signals for set plays.}
+\end{figure*}
+
+The gesture consists of raising one arm horizontally and sideways, parallel to the field line, while the other arm remains at rest.
+The raised arm is the one pointing towards the defending team's goal, i.e. the goal of the team that is \textit{not} taking the free kick.
+\todo[linecolor=gray,backgroundcolor=gray!25,bordercolor=gray]{
+  Remove the footnote after RC2025 has passed
+}
+\footnote{For reference, teams who participated in the 2023 Visual Referee Challenge
+  may notice that this the same as the "Kick-in" gesture from back then,
+  but with the opposite arm raised.
+}
+
+This gesture is made by the head referee and held for at least 5 seconds after the GameController signal is sent.
+The head referee is encouraged to seek the acknowledgement of the GameController operator and/or look at the field-facing monitor (if available) in order to determine the timing.
+
+The referee makes an effort to send the signal standing as close as possible to the T-junction joining the half-way line to the touchline opposite to the technical area.
+However, their actual positioning along the touchline may deviate slightly from this standard position, based on the current game situation and the timing of the gesture.
+\info[inline]{Remind the refs that the GC operator will take a few moments to click the button, giving them a little time to make a couple steps if needed.}
+
+Teams are encouraged to make an effort to familiarize with looking at the referee and interpreting the visual signal even in cases where such an action may not seem strictly necessary,
+as more information may be communicated in this way rather than via GameController packets in future years.
+
+\textbf{In Challenge Shield}, this visual signal does not need to be performed and the kicking team is communicated by the GameController.
```

Comment:
I think we think to think more about this. If there no change in GC for this, it may be hard for teams to do that without loosing time to execute the Set Play.

Reply:
I'm taking too long to write up an answer so maybe this is better left for the next meeting.

I'll just say that I am aware of this, and CS games are normally expected to be played without set-play gestures. The "does not need to" phrasing just means that a team requesting the gestures to be performed, for whatever reason they might have, is not technically forbidden.

Also, let me clarify that the todo notes are not intended to appear in the final rules document, but rather they are intended to be communicated informally.